src/raster/r.forcircular/r.forcircular.html

```diff
@@ -0,0 +1,64 @@
+<h2>DESCRIPTION</h2>
+
+Analyze and measure the level of sustainability of the forest-wood supply chain in a circular bioeconomy approach.
```

Comment:
```suggestion
<em>r.forcircular</em> analyzes and measures the level of sustainability of the forest-wood supply chain in a circular bioeconomy approach.
```

Reply:
As not everyone will read the reference paper, I'd suggest to include a bit longer explanation of how the module works, how parameters interact and any other particularity. Furthermore, an example and ideally a screenshot is desirable. Have a look at other addon manual pages for inspiration :)